### ISSUE TITLE
Areaは固定効果？ランダム効果？

### ISSUE THREAD

**Issue by ShotaNishijima** [ShotaNishijima/CPUE_workshop#13]
本筋からは逸れるのですが、標準化の際にareaを固定効果としてモデルに投入する特段の理由はあるのでしょうか。
まぐろ漁業の場合は地理的な影響はランダム効果として扱うか、エリアによりサイズ組成が異なる場合は別のモデルで標準化を実施して個別に資源評価モデルに投入することが多いです(VASTですとまとめて扱えるんでしたっけ)ので単純に興味を持ちました。

**Comment by ShotaNishijima** [comment]
欠損値がある場合はランダム効果だと便利ですよね。ランダム効果だとAreaの効果が正規分布にばらつく（でしかばらつかない）ことになるので、その制約によって予測性能が下がることはないのでしょうか？ランダム効果と固定効果の場合で結果は一致するんでしょうか？
VASTだとカテゴリーを分けて、複数のサイズをまとめてモデリングするんでしょうかね。

**Comment by ichimomo** [comment]
概念的な話ですが，やっぱり，海域の効果に興味があるかどうか，というところかなと思います．私は，どの海域でCPUEが潜在的に高いのか，というのが興味があるので，固定効果で推定したいです．年＊海域をランダム効果にすれば，年＊海域で欠損がある場合でも年効果が計算できて便利ですが，ほんとに欠損があって，そこが大事なところの場合，その部分の不確実性を見逃してしまうような．．

**Comment by otthii** [comment]
確かに、area shrinkingを見たいなど海域個別の動向を知りたい場合はエリアを固定効果として入れるべきですね。ただ、資源評価モデル用に一個のCPUE seriesを得るという観点ならば欠測値に強いランダム効果として扱った方が良いように思いますが、突き詰めるとVASTのように密度に時空間相関(と欠測値の内挿)を想定できるモデルが最も良いのかもしれません。
